central/blob/datastore/store/store.go

```diff
@@ -46,6 +51,15 @@ func wrapRollback(ctx context.Context, tx *pgPkg.Tx, err error) error {
 
 // Upsert adds a blob to the database
 func (s *storeImpl) Upsert(ctx context.Context, obj *storage.Blob, reader io.Reader) error {
+	if err := sac.VerifyAuthzOK(scopeChecker.WriteAllowed(ctx)); err != nil {
+		return err
+	}
+	// Augment permission because we require read permission internally
+	ctx = sac.WithGlobalAccessScopeChecker(ctx,
```

Comment:
Let's just do sac.WithAllAccess(ctx) for both these cases, but this is safe so 👍

Reply:
This is not needed to make it all access, so keep it as is?